src/ui/pages/work/partusage/PartUsageDialog.js

```diff
@@ -458,23 +612,62 @@ function PartUsageDialog(props) {
                                     formData.storeCode,
                                     formData.partCode
                                 ]}
-                                onChange={createOnChangeHandler(FORM.ASSET, FORM.ASSET_DESC, null, updateFormDataProperty, handleAssetChange)}
+                                onChange={createOnChangeHandler(FORM.ASSET, FORM.ASSET_DESC, null, updateFormDataProperty,
+                                    (asset) =>
+                                        runUiBlockingFunction(
+                                            handleAssetChange.bind(null, asset)
```

Comment:
Why are we using .bind on an arrow function instead of () => handleAssetChange(asset) ?

Reply:
no good reason I'll change all occurrences